cmd/tracee-ebpf/initialize/kernelconfig.go

```diff
@@ -0,0 +1,25 @@
+package initialize
+
+import (
+	"fmt"
+
+	"github.com/aquasecurity/libbpfgo/helpers"
+)
+
+func KernelConfig() (*helpers.KernelConfig, error) {
+	kernelConfig, err := helpers.InitKernelConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	// do not fail (yet ?) if we cannot init kconfig
+	kernelConfig.AddNeeded(helpers.CONFIG_BPF, helpers.BUILTIN)
+	kernelConfig.AddNeeded(helpers.CONFIG_BPF_SYSCALL, helpers.BUILTIN)
+	kernelConfig.AddNeeded(helpers.CONFIG_KPROBE_EVENTS, helpers.BUILTIN)
+	kernelConfig.AddNeeded(helpers.CONFIG_BPF_EVENTS, helpers.BUILTIN)
+	missing := kernelConfig.CheckMissing() // do fail if we found os-release file and it is not enough
+	if len(missing) > 0 {
+		return nil, fmt.Errorf("missing kernel configuration options: %s", missing)
```

Comment:
This changes the semantics of how we treat this error.
Before it was a critical error, which won't allow to continue the execution of tracee, now it will just emit debug messages, just like the error returned by InitKernelConfig (but should be treaten differently)

Reply:
True, i'll fix that.